src/client/store/geo/mosaic/hooks/mosaic.ts

```diff
@@ -10,8 +10,7 @@ export const useMosaicUrlTemplateData = (): MosaicUrlTemplateData | undefined =>
 
 export const useMosaicOptions = (): MosaicOptions => useAppSelector((state) => MosaicSelectors.getOptions(state))
 
-export const useMosaicSelected = (): boolean | undefined =>
-  useAppSelector((state) => MosaicSelectors.getSelected(state))
+export const useMosaicSelected = (): boolean => useAppSelector((state) => MosaicSelectors.getSelected(state))
```

Comment:
Not sure if this works?
```suggestion
export const useMosaicSelected = (): boolean => useAppSelector(MosaicSelectors.getSelected)
```

Reply:
Yes that works, thanks!